docs/getting-started.md

```diff
@@ -3,6 +3,7 @@
 - [Install](#install)
 - [Create an API](#create-an-api)
 - [Access an API](#access-an-api)
+- [Publish an API](#publish-an-api-to-the-tyk-portal)
```

Comment:
I would caveat this with "Licensed" installation only. Something like:

![Screenshot 2022-03-08 at 13 52 21](https://user-images.githubusercontent.com/1465130/157251309-87703da6-94d3-4131-a599-cbedb0f739c9.png)
.

Reply:
@asoorm I have updated the file following your advice.